WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressValidationViewController.swift

```diff
@@ -0,0 +1,309 @@
+import UIKit
+import Yosemite
+
+final class ShippingLabelAddressValidationViewController: UIViewController {
+
+    @IBOutlet private weak var tableView: UITableView!
+    @IBOutlet private weak var confirmButton: UIButton!
+
+    private let viewModel: ShippingLabelAddressValidationViewModel
+
+    /// Needed to scroll content to a visible area when the keyboard appears.
+    ///
+    private lazy var keyboardFrameObserver = KeyboardFrameObserver(onKeyboardFrameUpdate: { [weak self] frame in
+        self?.handleKeyboardFrameUpdate(keyboardFrame: frame)
+    })
+
+    /// Init
+    ///
+    init(addressVerification: ShippingLabelAddressVerification) {
+        viewModel = ShippingLabelAddressValidationViewModel(addressVerification: addressVerification)
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureNavigationBar()
+        configureMainView()
+        configureTableView()
+        registerTableViewCells()
```

Comment:
nit: we probably don't need to call this since it's already called in `configureTableView`?